proto/stakeibc/genesis.proto

```diff
@@ -27,7 +21,6 @@ message GenesisState {
   // stores a map from hostZone base denom to hostZone
   map<string, string> denomToHostZone = 9;
   repeated EpochTracker epochTrackerList = 10 [(gogoproto.nullable) = false];
-  repeated PendingClaims pendingClaimsList = 11 [(gogoproto.nullable) = false];
   // this line is used by starport scaffolding # genesis/proto/state
-  reserved 3;
+  reserved 3, 11;
```

Comment:
I'm pretty sure this would be fine, but there's some tail risk this breaks something and removing these is low upside. going to leave them in to respect proto invariants (easier to consistently leave them in than to reason about when to remove them)